src/helpers/genericArticleHelper.js

```diff
@@ -44,35 +44,53 @@ const filter = (title, tag, author, keyword) => {
   if (!tag) delete queryParams.query['$tag.category$'];
 
   // eslint-disable-next-line no-nested-ternary
-  const filterQuery = (keyword) ? { ...queryParams.keyword } : author ? { ...queryParams.author }
-    : { ...queryParams.query };
+  const filterQuery = keyword
```

Comment:
Please refactor this piece of code to something more readable. We want to avoid nested ternary operations at all times

Reply:
Done.